source/includes/pluggable-scm/_02-create.md.erb

```diff
@@ -0,0 +1,89 @@
+## Create a SCM object
+
+```shell
+$   curl 'http://ci.example.com/go/api/admin/scms' \
+  -u 'username:password' \
+  -H 'Accept:application/vnd.go.cd.v1+json' \
+  -H 'Content-Type:application/json' \
+  -X POST -d '{
+  "id": "scm-id",
+  "name": "foobar",
+  "auto_update": true,
+  "plugin_configuration": {
+    "id": "github.pr",
+    "version": "1"
+  },
+  "configuration": [
+    {
+      "key": "url",
+      "value": "git@github.com:sample/example.git"
+    },
+    {
+      "key": "username",
+      "value": "admin",
+      "secure": true
+    },
+    {
+      "key": "password",
+      "encrypted_value": "aSdiFgRRZ6A=",
+      "secure": true
+    }
+  ]
+ }'
+```
+
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "http://ci.example.com/go/api/admin/scms/foobar"
+    },
+    "doc": {
+      "href": "http://api.go.cd/#scms"
+    }
+  },
+  "id": "scm-id",
+  "name": "foobar",
+  "auto_update": true,
+  "plugin_configuration": {
+    "id": "github.pr",
+    "version": "1"
+  },
+  "configuration": [
+    {
+      "key": "url",
+      "value": "git@github.com:sample/example.git"
+    },
+    {
+      "key": "username",
+      "encrypted_value": "50TVtMW3cBU="
+    },
+    {
+      "key": "password",
+      "encrypted_value": "aSdiFgRRZ6A="
+    }
+  ]
+}
+
+```
+
+Create a global SCM object.
+
+
+<%= available_since('16.6.0') %>
+
+<p class='http-request-heading'>HTTP Request</p>
+
+`POST /go/api/admin/scms`
+
+<p class='http-request-return-description'>Returns</p>
+
+A new SCM object.
```

Comment:
Please link to a section that describes the SCM object and it's nested properties, you'll find examples in other places (agents, pipeline configs etc).

Work with @maheshp so you're not duplicating efforts :)